src/auth.d.ts

```diff
@@ -494,6 +505,18 @@ export namespace admin.auth {
      */
     iss: string;
 
+    /**
+     * The phone number corresponding to the user who the ID token belonged to,
+     * if available.
+     */
+    phone_number?: string;
+
+    /**
+     * The photo URL corresponding to the user who the ID token belonged to,
```

Comment:
Suggest "The photo URL for the user to whom the ID token belonged..."

Reply:
done